draft-packages/stories/TitleBlockZen.stories.tsx

```diff
@@ -156,15 +152,13 @@ export const WithDefaultTag = () => (
   />
 )
 
-WithDefaultTag.story = {
-  name: "With Default Survey Status (Tag)",
-}
+WithDefaultTag.storyName = "With Default Survey Status (Tag)"
 
 export const AdminWithDefaultTag = () => (
   <TitleBlockZen
     variant="admin"
     title="Page title"
-    surveyStatus={{ text: "Due July 8, 2030", status: "default" }}
+    surveyStatus={{ text: "Due July 8, 2030", status: "draft" }}
```

Comment:
This one should actually stay `default` because it was added recently by @vicchirino in a PR that added a `default` variant to `surveyStatus`.

Reply:
Oh, my bad! It was throwing type errors in my editor.